das/store.go

```diff
@@ -0,0 +1,114 @@
+package das
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/namespace"
+	"golang.org/x/net/context"
+)
+
+var (
+	storePrefix   = datastore.NewKey("das")
+	checkpointKey = datastore.NewKey("checkpoint")
+)
+
+type checkpointStore interface {
```

Comment:
Why does this have to be an interface? Why not use dataStore directly? I understand the need for testing, but there is an in-mem ds that you can use.

Reply:
Yeah interface was mainly introduced to create mock independent of third party package functionality on a cost of extra abstraction. Otherwise would be store->load->assert semantic. But tests are less important then main code, I think it is good idea to get rid of it. Thank you!